Scala/Scala.sublime-syntax

```diff
@@ -338,7 +338,7 @@ contexts:
         1: storage.type.function.scala
         2: entity.name.function.scala
       push: function-type-parameter-list
-    - match: '\b(case\s+)?(class|trait|object)(\s+({{id}}))'
+    - match: '\b(case\s+)?(class|trait|object)\s+({{id}})'
```

Comment:
`\b(?:(case)\s+)?` - to ensure the word boundaries of `case` are respected as well.

Reply:
@djspiewak I think the capture groups need to be tweaked as @deathaxe mentioned so that the space after case doesn't get the same scope name.